File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// Nationalize SDK utility: result_body

class NationalizeResultBody
{
    public static function call(NationalizeContext $ctx): ?NationalizeResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/rb/utility/result_body.rb
# Nationalize SDK utility: result_body
module NationalizeUtilities
  ResultBody = ->(ctx) {
    response = ctx.response
    result = ctx.result
    if result && response && response.json_func && response.body
      result.body = response.json_func.call
    end
    result
  }
end
